src/typeDefs/queries.ts

```diff
@@ -108,6 +108,8 @@ export const queries = gql`
 
     users(where: UserWhereInput, orderBy: UserOrderByInput): [User] @auth
 
+    usersByTag(tagId: ID!): [User] @auth
```

Comment:
Remove this, this can easily be queried using `tag.usersAssignedTo` relation connection.